frontend/plugins/frontline_ui/src/modules/inbox/conversations/components/ConversationDisplay.tsx

```
@@ -0,0 +1,46 @@
+    Split view
+  </ToggleGroup.Item>
+  <ToggleGroup.Item
+    value="list"
+    className="h-14 flex-col flex-auto gap-1 [&_svg]:size-5 text-muted-foreground data-[state=on]:bg-muted data-[state=on]:text-foreground"
+    variant="default"
+  >
+    <IconList />
+    List view
+  </ToggleGroup.Item>
+</ToggleGroup>; */
+}
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.

(lint/complexity/noUselessLoneBlockStatements)